ui/admin/tests/acceptance/targets/list-test.js

```diff
@@ -269,30 +264,6 @@ module('Acceptance | targets | list', function (hooks) {
     assert.dom(selectors.TABLE_SESSIONS_ID(instances.session.id)).isVisible();
   });
 
-  test('targets table is sorted by `created_time` descending by default', async function (assert) {
```

Comment:
I removed this test since right now by default I don't add a sort on `created_time` to every request since it _should_ already be sorted in that order by backend.

My main thinking on this is if we do add the default sort to every request, that means we likely need to add an index to `created_time` for every resource as well which I would like to avoid since there isn't a big benefit besides ensuring the right order on the off chance that it got out of order at some point while the main con is more disk space usage for all the indexes (and maybe slower writes).

Let me know your thoughts on this.

Reply:
I also wanted to point out that I changed the index to a composite index here: 
https://github.com/hashicorp/boundary-ui/blob/b64228d96680458dc8dbe7a5c8abec49626e9324/addons/api/addon/workers/utils/schema.js#L33

With two individual indexes (one on `scope_id` and one on `created_time`), only the `scope_id` was getting used to filter and the ORDER BY won't end up using the `created_time` index. The composite index has the benefit of still being used for just a `scope_id` WHERE clause too so it won't be any worse.